packages/build/tests/monitor/tests.js

```diff
@@ -124,3 +125,21 @@ test.serial('When an error has a `normalizedMessage` property, its value is used
 
   stub.restore()
 })
+
+test('Error messages are normalized', (t) => {
+  const cases = [
+    [
+      'functionsBundling Command failed with exit code 1: /opt/build/repo/.netlify/plugins/deno-cli/deno bundle --import-map=data:application/json;base64,eyJpbXBvcnRzIjp7Im5ldGxpZnk6ZWRnZSI6Imh0dHBzOi8vZWRnZS5uZXRsaWZ5LmNvbS92MS9pbmRleC50cyJ9fQ== --quiet /tmp/edge-62e3a15d47b3340008b4b904/e6c2a079-8110-438d-b03d-ec2d2b6ca178-pre.js /tmp/edge-62e3a15d47b3340008b4b904/e6c2a079-8110-438d-b03d-ec2d2b6ca178.js',
+      'functionsBundling Command failed with exit code 0: /file/path bundle --import-map=dataURI --quiet /file/path /tmp/edghex/hex.js',
+    ],
+
+    [
+      'Bundling of function "server" failed\nBuild failed with 1 error: .netlify/functions-internal/server/node_modules/ufo/package.json:41:1: ERROR: Expected end of file but found ","',
+      'Bundling of function "functionName" failed\nBuild failed with 0 error: /file/path ERROR: Expected end of file but found ","',
+    ],
+  ]
+
+  cases.forEach(([original, expected]) => {
```

Comment:
this test means that the two cases in the array normalize to the same thing, do I understand correctly?

Reply:
No. Each element in the outer array is a test case. Each test case has two elements: the original value and the expected result.

I've added a comment in https://github.com/netlify/build/pull/4396/commits/afb07f637e1db913558383e1be4db508163a8fc5, hopefully that makes it clearer.